docs/source/programs_and_gates.rst

```diff
@@ -685,49 +702,3 @@ All of the above methods will produce the same output:
 
    X 0
    Y 1
-
-The ``pyquil.parser`` submodule provides a front-end to other similar parser
-functionality.
-
-
-Fixing a Mistaken Instruction
------------------------------
-
-If an instruction was appended to a program incorrectly, you can pop it off.
-
-.. testsetup:: mistake
-
-   from pyquil import Program
-   from pyquil.gates import X, Y
-
-.. testcode:: mistake
-
-    p = Program(X(0), Y(1))
-    print(p)
-
-    print("We can fix by popping:")
-    p.pop()
-    print(p)
-
-.. testoutput:: mistake
-
-    X 0
-    Y 1
-    We can fix by popping:
-    X 0
-
-QPU-allowable Quil
-~~~~~~~~~~~~~~~~~~
```

Comment:
Shying away from making any assertions about what does or doesn't work on a QPU in pyQuil docs. Should we try and replicate this somewhere in the Rigetti docs?